src/dotnet/commands/dotnet-cache/Program.cs

```diff
@@ -71,10 +74,43 @@ public static int Run(string[] args)
                 cache.ExtraMSBuildArguments = app.RemainingArguments;
                 cache.ProjectArgument = projectArgument.Value();
                
-                return cache.Execute();
+                return 0;
             });
 
-            return app.Execute(args);
+            int exitCode = app.Execute(args);
```

Comment:
I think we would need some kind of static virtual methods to put FromArgs in a base class - I think this would be to hacky to do. @jonsequitur @piotrpMSFT - any ideas how to implement that without adding much complexity? T4 template which would generate this for each instance perhaps?